app/views/foods/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: food) do |form| %>
+  <% if food.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(food.errors.count, "error") %> prohibited this food from being saved:</h2>
+
+      <ul>
+        <% food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ] If you can make your error message to be a bit user-friendly, that will be nice
![Screenshot from 2023-10-12 17-33-19](https://github.com/batoolfatima2135/Recipe-app/assets/66097371/263b7403-c4c1-486f-a369-01f2ce5ad94b)